milabench/cli.py

```diff
@@ -164,6 +169,9 @@ def _get_multipack(
     overrides={},
     return_config=False,
 ):
+    if system_config_path is None:
```

Comment:
We will make the system file only required for multi-node.
We should try to make single node bench as simple as possible, multi-node is more rare.